python/TITLE CASE

```diff
@@ -0,0 +1,13 @@
+def title_case(name):
+      size=len(name)
+      a=" "
+      for i in range(size):
+            if name[i]="" or name[i]==".":
```

Comment:
hey @Priyanshu729 we're getting errors

- change `if name=""` to if name==""`
- and correct the indentations which is `4 spaces`. also include a screenshot of the output

Reply:
i will solve that error can i resend it??